app/navigation/Navigator.scala

```diff
@@ -27,24 +27,26 @@ import queries.CategorisationQuery
-    case UkimsNumberPage        => _ => routes.HasNirmsController.onPageLoad(NormalMode)
-    case HasNirmsPage           => navigateFromHasNirms
-    case NirmsNumberPage        => _ => routes.HasNiphlController.onPageLoad(NormalMode)
-    case HasNiphlPage           => navigateFromHasNiphl
-    case NiphlNumberPage        => _ => routes.CyaCreateProfileController.onPageLoad
-    case CreateRecordStartPage  => _ => routes.TraderReferenceController.onPageLoad(NormalMode)
-    case TraderReferencePage    => _ => routes.UseTraderReferenceController.onPageLoad(NormalMode)
-    case UseTraderReferencePage => navigateFromUseTraderReference
-    case GoodsDescriptionPage   => _ => routes.CountryOfOriginController.onPageLoad(NormalMode)
-    case CountryOfOriginPage    => _ => routes.CommodityCodeController.onPageLoad(NormalMode)
-    case CommodityCodePage      => _ => routes.HasCorrectGoodsController.onPageLoad(NormalMode)
-    case HasCorrectGoodsPage    => navigateFromHasCorrectGoods
-    case p: AssessmentPage      => navigateFromAssessment(p)
-    case AdviceStartPage        => _ => routes.NameController.onPageLoad(NormalMode)
-    case NamePage               => _ => routes.EmailController.onPageLoad(NormalMode)
-    case EmailPage              => _ => routes.CyaRequestAdviceController.onPageLoad
-    case _                      => _ => routes.IndexController.onPageLoad
+    case ProfileSetupPage         => _ => routes.UkimsNumberController.onPageLoad(NormalMode)
+    case UkimsNumberPage          => _ => routes.HasNirmsController.onPageLoad(NormalMode)
+    case HasNirmsPage             => navigateFromHasNirms
+    case NirmsNumberPage          => _ => routes.HasNiphlController.onPageLoad(NormalMode)
+    case HasNiphlPage             => navigateFromHasNiphl
+    case NiphlNumberPage          => _ => routes.CyaCreateProfileController.onPageLoad
+    case CreateRecordStartPage    => _ => routes.TraderReferenceController.onPageLoad(NormalMode)
+    case TraderReferencePage      => _ => routes.UseTraderReferenceController.onPageLoad(NormalMode)
+    case UseTraderReferencePage   => navigateFromUseTraderReference
+    case GoodsDescriptionPage     => _ => routes.CountryOfOriginController.onPageLoad(NormalMode)
+    case CountryOfOriginPage      => _ => routes.CommodityCodeController.onPageLoad(NormalMode)
+    case CommodityCodePage        => _ => routes.HasCorrectGoodsController.onPageLoad(NormalMode)
+    case HasCorrectGoodsPage      => navigateFromHasCorrectGoods
+    case p: AssessmentPage        => navigateFromAssessment(p, NormalMode)
+    case AdviceStartPage          => _ => routes.NameController.onPageLoad(NormalMode)
+    case NamePage                 => _ => routes.EmailController.onPageLoad(NormalMode)
+    case EmailPage                => _ => routes.CyaRequestAdviceController.onPageLoad
+    case HasSupplementaryUnitPage => navigateFromHasSupplementaryUnit
```

Comment:
can you change the order of this so the assessment journey is all together

Reply:
Done